src/PhoneNumber/PrefixSelector/PrefixSelector.js

```diff
@@ -6,24 +6,38 @@ import classNames from 'classnames';
 import scrollIntoView from 'dom-scroll-into-view';
 import Option from './Option';
 import Options from './Options';
+import debounce from '../../utils/debounce';
 
 const INITIAL_INDEX = -1;
-const KEYS = [13, 27, 38, 40];
-const [ENTER, ESC, ARROW_UP, ARROW_DOWN] = KEYS;
-const styles = {
-  fakeInput: {
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    width: '100%',
-    height: '100%',
-    opacity: 0
+const KEYS = [13, 27, 38, 40, 9];
+const WAIT_TIME = 200;
+const [ENTER, ESC, ARROW_UP, ARROW_DOWN, TAB] = KEYS;
+const EMPTY_STRING = '';
+const getA11yStatusMessage = ({ isOpen, options, selectedOption }) => {
```

Comment:
Wouldn't make sense to move it to a utilities class if this is reused in several components?

Reply:
Yes carlos, I'll open a concern so we can improve it in flycomponents and also in Payex which is repeated in several places too.